runtime/jcl/common/criusupport.c

```diff
@@ -283,16 +318,25 @@ Java_org_eclipse_openj9_criu_CRIUSupport_checkpointJVMImpl(JNIEnv *env,
 		resultType = vm->criuSupportSuccess;
 
 releaseExclusive:
+		vmFuncs->releaseExclusiveVMAccess(currentThread);
+closeWorkDirFD:
+		if (0 != close(workDirFD)) {
+			systemReturnCode = errno;
+			resultType = vm->criuSupportJVMRestoreFailure;
```

Comment:
This is the wrong result type if the failure occurred before calling `criu_dump()`.